src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+
```

Comment:
Why is the class empty? You need to store the Block number within the Block class and provide appropriate methods to set/access it.